client/state/happychat/test/selectors.js

```diff
@@ -276,6 +277,17 @@ describe( 'selectors', () => {
 	} );
 
 	describe( '#getGroups()', () => {
+		let _window; // Keep a copy of the original window if any
```

Comment:
We need that because `isATEnabled` is checking for window object to prevent ssr issues, whatever that means.